.babelrc

```diff
@@ -0,0 +1,15 @@
+{
+  "presets": [
+    [
+      "env", {
+        "targets": {
+          "electron": "1.6.6"
+        }
+      }
+    ],
+    "react"
+  ],
+  "only": [
+    "lib/**/*.jsx"
```

Comment:
In the `babel` Makefile target it seems you also process the `tests` directory - does that need to be added here too?

Reply:
Yeah, good catch.